_includes/utilities/responsive-variants.html

```diff
@@ -63,21 +63,17 @@ <h4 class="font-lang-2xs margin-top-2 margin-bottom-05">Output</h4>
     </tbody>
   </table>
 
-  <p class="utility-text margin-top-4">Set whether a utility family outputs with responsive variants by setting the value of <code>$[utility_family]-settings.responsive</code> in your <a href="{{ site.baseurl }}/documentation/settings">USWDS settings configuration</a>. When modifying any settings stored as a map, you need to set <em>all</em> elements of the map, not just the element you wish to change.</p>
+  <p class="utility-text margin-top-4">Set whether a utility family outputs with responsive variants by setting the value of <code>$[utility_family]-settings.responsive</code> in your <a href="{{ site.baseurl }}/documentation/settings">USWDS settings configuration</a>. When modifying any settings stored as a map, undefined elements of the map will take its default value.</p>
```

Comment:
Having trouble thinking clearly atm, but this feels like better English?
```suggestion
  <p class="utility-text margin-top-4">Set whether a utility family outputs with responsive variants by setting the value of <code>$[utility_family]-settings.responsive</code> in your <a href="{{ site.baseurl }}/documentation/settings">USWDS settings configuration</a>. When modifying settings stored as a map, undefined map elements will keep their default values.</p>
```